heracal/omni.py

```diff
@@ -532,6 +565,102 @@ def from_fits(filename, pols=None, bls=None, ants=None, verbose=False):
                             
     return meta, gains, v, x
 
+def write_uvdata_vis(filename, aa, m, v, xtalk=False, returnuv=True):
+    '''
+    Given meta information and visibilities, write out a uvfits file.
+    filename: filename to write uvfits object.(str)
+    aa      : aipy antenna array object (object)
+    m       : dictionary of information (dict)
+    v       : dictionary of visibilities with keys antenna pair and pol (dict)
+    xtalk   : visibilities given are xtalk visibilities. (bool)
+    returnuv: return uvdata object. If returned, file is not written. (bool)
```

Comment:
This is not the same argument/return documentation format as used in `minVreds`.